tests/utils/basic.inc

```diff
@@ -8,5 +8,7 @@ define('DATABASE_NAME', getenv('MONGODB_DATABASE') ?: 'phongo');
 define('COLLECTION_NAME', makeCollectionNameFromFilename($_SERVER['SCRIPT_FILENAME']));
 define('NS', DATABASE_NAME . '.' . COLLECTION_NAME);
 define('SSL_DIR', realpath(getenv('SSL_DIR')));
-define('CSFLE_KEY_VAULT_NS', 'encryption.__keyVault');
+define('CSFLE_KEY_VAULT_DATABASE_NAME', 'encryption');
+define('CSFLE_KEY_VAULT_COLLECTION_NAME', '__keyVault');
+define('CSFLE_KEY_VAULT_NS', CSFLE_KEY_VAULT_DATABASE_NAME . '.' . CSFLE_KEY_VAULT_COLLECTION_NAME);
```

Comment:
Splitting these was necessary to utilize `skip_if_not_clean` in tests. I originally created a `skip_if_ns_not_clean` method but this seemed more straightforward.